packages/build-info/src/frameworks/hydrogen.ts

```diff
@@ -1,25 +1,67 @@
-import { BaseFramework, Category, Framework } from './framework.js'
+import { Accuracy, BaseFramework, Category, DetectedFramework, Framework } from './framework.js'
+
+const CLASSIC_COMPILER_CONFIG_FILES = ['remix.config.js']
+const CLASSIC_COMPILER_DEV = {
+  command: 'remix dev --manual -c "netlify dev"',
+}
+const CLASSIC_COMPILER_BUILD = {
+  command: 'remix build',
+  directory: 'public',
+}
+
+const VITE_CONFIG_FILES = [
+  'vite.config.js',
+  'vite.config.mjs',
+  'vite.config.cjs',
+  'vite.config.ts',
+  'vite.config.mts',
+  'vite.config.cts',
+]
+const VITE_DEV = {
+  command: 'shopify hydrogen dev',
+  port: 5173,
+}
+const VITE_BUILD = {
+  command: 'remix vite:build',
```

Comment:
It might be useful to explain here why this is necessary instead of using `shopify hydrogen build` (I guess it's a temporary fix until shopify/hydrogen#2497 and shopify/hydrogen#2497 are resolved)